interfaces/mount/entry.go

```diff
@@ -103,3 +119,28 @@ func (e Entry) String() string {
 	return fmt.Sprintf("%s %s %s %s %d %d",
 		name, dir, fsType, options, e.DumpFrequency, e.CheckPassNumber)
 }
+
+// ParseEntry parses a fstab-like entry.
+func ParseEntry(s string) (Entry, error) {
+	var e Entry
+	fields := strings.Fields(s)
```

Comment:
After trying with `Scanner` API I now did this with the trivial `strings.FieldsFunc` method. Have a look please.